icu4c/source/common/unicode/utypes.h

```diff
@@ -384,6 +384,81 @@ typedef double UDate;
 #define U_TOOLUTIL_API U_IMPORT
 #endif
 
+/**
+ * \def U_DATA_API_CLASS
+ * Set to export library symbols from inside the stubdata library,
+ * and to import them from outside, to be used on a class.
+ * @draft ICU 78
```

Comment:
> There's a testtagsguards.sh failure in clang-release-build-and-test, what's the correct solution for that?

You need to surround the new @draft definitions with
```
#ifndef U_HIDE_DRAFT_API
#endif  // U_HIDE_DRAFT_API
```

Or, in this case, since we can't compile without these, “use the force” -- U_FORCE_HIDE_DRAFT_API

Reply:
Done.